src/App.tsx

```diff
@@ -21,14 +66,23 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading && (
+                <Loader />
+              )}
+
+              {todos.length > 0 && (
+                <TodoList
+                  todos={visibleTodos}
+                />
```

Comment:
It can be in one line. But it's up to you
```suggestion
                <TodoList todos={visibleTodos} />
```